libs/elements/cards/src/lib/elements-cards.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ElewaProjectItemCardComponent } from './elewa-project-item-card/elewa-project-item-card.component';
+
+@NgModule({
+  imports: [CommonModule],
+  declarations: [ElewaProjectItemCardComponent],
```

Comment:
There are two modules here. The buttons module is the only module that was needed. Cards should just be a folder. So path should be `cards/layout/buttons` with buttons as the module

Reply:
Technical Analysis
This component will be created in libs/elements/cards (cards is a module) on the instructions. Am now confused.